src/message/rpc.proto.js

```diff
@@ -1,3 +1,6 @@
+'use strict'
+
+module.exports = `
 message RPC {
```

Comment:
remove the newline after the first tick, there is a bug in uglify that throw on this https://github.com/mishoo/UglifyJS2/pull/1353